### DISCUSSION TITLE
Owls and other nocturnal birds

### DISCUSSION THREAD

**Discussion by jmtmp** [Nachtzuster/BirdNET-Pi#265] [Ideas]
I would like to return to the discussion about nocturnal bird detection, which can be improved by significantly lowering the "Species Occurrence Frequency Threshold" parameter. However, this setting may have the unintended effect of incorrectly detecting rare bird species during the day.
Wouldn't a better solution be to refine the calculation of the "historically submitted eBird checklist" not only for a given location and week, but also for a given time (hour or night/day)?

**Comment by alexbelgium** [comment]
Hi, this is indeed a known issue. However we can't do anything at our level : this must be done at the level of the model itself here https://github.com/kahst/BirdNET-Analyzer/issues
This has been already reported to them but we don't know if/when it will be improved https://github.com/kahst/BirdNET-Analyzer/issues/296

**Reply by no-username-found** [reply]
I've tried using the whitelist_species_list.txt for this, however it was not implemented. I have a pull request open now that fixes it. At my side, the owls I whitelisted are detected.
